cli.js

```diff
@@ -48,6 +49,7 @@ const cli = meow(`
 	  -c, --crop               Crop to the set height
 	  -d, --delay=<seconds>    Delay screenshot capture
 	  --filename=<template>    Custom filename
+	  --overwrite              Overwrite file if it exists
```

Comment:
Needs to be documented in the readme too.

Reply:
Fixed in https://github.com/sindresorhus/pageres-cli/pull/5/commits/97ff81b012e173ef97bc2ea53a00861defb94567